tests/python/direct/test_cutlass_nvfp4_gemm.py

```diff
@@ -7,9 +7,10 @@
 import torch
 from nvfuser_direct import nvf_cutlass
 
-if torch.cuda.get_device_capability() < (10, 0):
+compute_cap = torch.cuda.get_device_capability()
+if compute_cap < (10, 0) or compute_cap >= (12, 0):
```

Comment:
Will this work for GB300? I think its compute capability is `10.3`.

Reply:
It should, this would allow compute capabilities in the range of [10.0, 12.0)